src/organizations/components/OrgProfileTab/index.tsx

```diff
@@ -43,7 +43,7 @@ const OrgProfileTab: FC = () => {
       if (
         !me.quartzMe.regionCode ||
         !me.quartzMe.regionName ||
-        !identity.org.provider
+        !identity.currentIdentity.org.provider
```

Comment:
FYI - these are just updates resulting from a structural refactor to the one state in the app that currently uses `identity` directly, rather than through the conversion to the legacy `quartzMe` state.